ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java

```diff
@@ -641,7 +641,8 @@ private QueryErrorClassifier getConfiguredQueryErrorClassifier(
       final String applicationId
   ) {
     final QueryErrorClassifier userErrorClassifiers = new MissingTopicClassifier(applicationId)
-        .and(new AuthorizationClassifier(applicationId));
+        .and(new AuthorizationClassifier(applicationId))
+        .and(new KsqlFunctionClassifier(applicationId));
```

Comment:
I want to make sure that we get buy-in on the new classifier; I definitely like the idea!